CHANGELOG.md

```diff
@@ -59,9 +59,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - In Traffic Portal, all tables now include a 'CSV' link to enable the export of table data in CSV format.
 - Pylint configuration now enforced (present in [a file in the Python client directory](./traffic_control/clients/python/pylint.rc))
 - Added an optional SMTP server configuration to the TO configuration file, api now has unused abilitiy to send emails
-- In Traffic Portal, added the ability to create, view and delete server capabilities as associate those server capabilities with servers.
 - Traffic Monitor now has "gbps" calculated stat, allowing operators to monitor bandwidth in Gbps.
 - Added an API 1.4 endpoint, /api/1.4/deliveryservices_required_capabilities, to create, read, and delete associations between a delivery service and a required capability.
+- In Traffic Portal, added the ability to create, view and delete server capabilities and associate those server capabilities with servers and delivery services. See [blueprint](https://github.com/apache/trafficcontrol/blob/master/blueprints/server-capabilitites.md)
```

Comment:
Bit of a nit, but links like this should always be relative (i.e. `./blueprints/server-capabilities.md`) so that they always point to the version at which they're being read.

Reply:
makes sense